teensy/main/main.ino

```diff
@@ -81,28 +93,41 @@ int count = 0;
 void print_all_sensors() {
   //prints a single string containing all sensor values
   //to be decoded by python script
-
+  String output = "";
   //expect precision issues to # of digits
   //python script currently expects 2 digits per datum
 
   //GPS
   Serial1.print("^latitude:");
   Serial1.println(myGps->get_latitude());
+  output.concat("^latitude:");
+  output.concat(myGps->get_latitude());
```

Comment:
Can we each data prefix and its data with a comma or semicolon?

Reply:
so you want something like latitude:xx;longitude:xx;...? and this is only for the SD card or even to send to the ESP?

Reply:
Yes, and just the SD card for now is fine